ebd/eapi/8/phase.bash

```diff
@@ -5,3 +5,15 @@ usev() {
 	fi
 	return 1
 }
+
```

Comment:
PKGCORE_BANNED_FUNCS should be used instead of explicitly defining stub functions.

Reply:
This works for useq but not hasq, hasv. If I create `ebd/eapi/8/depend.bash`, it does not seem to be used at all. If I add them in `global.bash` or `phase.bash`, pkgcore complains:

```
EAPI 8: undefined banned func: 'hasq'

Reply:
> This works for useq but not hasq, hasv. If I create `ebd/eapi/8/depend.bash`, it does not seem to be used at all. If I add them in `global.bash` or `phase.bash`, pkgcore complains:
> 
> ```
> EAPI 8: undefined banned func: 'hasq'
> ```

That's fixed now in git so all function names should be able to be placed in the PKGCORE_BANNED_FUNCS array.

Reply:
Ok, thanks.